bugbug/repository.py

```diff
@@ -515,7 +515,6 @@ def get_summary_metrics(obj, metrics_space):
         obj["halstead_N2_min"] = min(obj["halstead_N2_min"], metrics["halstead"]["N2"])
         obj["halstead_n1_min"] = min(obj["halstead_n1_min"], metrics["halstead"]["n1"])
         obj["halstead_N1_min"] = min(obj["halstead_N1_min"], metrics["halstead"]["N1"])
-
```

Comment:
Unrelated change. Could you please revert it?

Reply:
Initially, I was confused looking at this issue, I think it has already been  [done](https://github.com/mozilla/bugbug/pull/2372/files) but not linked to the [open issue](https://github.com/mozilla/bugbug/issues/1883).

That was why I picked it up. I think we can close the PR. Please let me know your thoughts on this.

Reply:
Sorry for that. I will close this PR. Feel free to select another issue. Please let me know if you need any help.